contracts/governance/extensions/GovernorOverrideDelegateVote.sol

```diff
@@ -0,0 +1,181 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {GovernorVotes} from "./GovernorVotes.sol";
+import {IVotes} from "../utils/IVotes.sol";
+import {IERC5805} from "../../interfaces/IERC5805.sol";
+import {SafeCast} from "../../utils/math/SafeCast.sol";
+import {Time} from "../../utils/types/Time.sol";
+import {VotesOverridable} from "../utils/VotesOverridable.sol";
+
+/**
+ * @dev Extension of {Governor} which enables delegatees to override the vote of their delegates. This module requires a
+ * token token that inherits `VotesOverridable`.
+ */
+abstract contract GovernorOverrideDelegateVote is GovernorVotes {
+    /**
+     * @dev Supported vote types. Matches Governor Bravo ordering.
+     */
+    enum VoteType {
+        Against,
+        For,
+        Abstain
+    }
+
+    struct VoteReceipt {
+        uint8 support;
```

Comment:
if this stores "support + 1", maybe we should not name it suppor, but something different.

most basic name would be `supportPlusOne`, but it could be anything that makes the distinction clear.